eva/expression/comparison_expression.py

```diff
@@ -68,11 +62,8 @@ def evaluate(self, *args, **kwargs):
             return Batch.compare_contains(lbatch, rbatch)
         elif self.etype == ExpressionType.COMPARE_IS_CONTAINED:
             return Batch.compare_is_contained(lbatch, rbatch)
-        else:
```

Comment:
Same as above. Am I missing something? How do we take care of such cases?

Reply:
Added `assert`